### DISCUSSION TITLE
Shadow only render pass ?

### DISCUSSION THREAD

**Discussion by negicless** [bnpr/Malt#502] [Q&A]
Hey,  I am currently working on a project that needs a separate shadow pass to do composition in aftereffect,  since there is no built-in soft shadow support right now, is there a way to render only shadows?

**Comment by pragma37** [comment]
There are no built-in Render Passes types in Malt, but you can do it with custom outputs (AOVs). See https://github.com/bnpr/Malt/discussions/91#discussioncomment-2891018.

For getting only shadows you can do something like this:
![imagen](https://github.com/bnpr/Malt/assets/36610999/fb539e02-df4f-4e2a-adbc-4587661d27bd)

[shadow-aov.zip](https://github.com/bnpr/Malt/files/11807590/shadow-aov.zip)

**Reply by pragma37** [reply]
If you're going to move the result to After Effects anyway you can simply use the shadow pass from Cycles too.
